dev/com.ibm.ws.security.openidconnect.server/resources/com/ibm/ws/security/openidconnect/server/internal/resources/OidcServerMessages.nlsprops

```diff
@@ -223,10 +223,18 @@ BACKCHANNEL_LOGOUT_REQUEST_EXCEPTION.useraction=Ensure that the back-channel log
 # NOTE: Start of new NLS prefix block: CWWKS1950 - CWWKS1999
 #
 
-BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED=CWWKS1950E: The {0} OpenID Connect provider encountered an error while submitting back-channel logout requests: {1}
-BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED.explanation=Sending requests to the back-channel logout URIs was interrupted, or another error occurred while scheduling the requests for execution.
+BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED=CWWKS1950E: The {0} OpenID Connect provider encountered an error on submitting back-channel logout requests: {1}
+BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED.explanation=Requests sent to the back-channel logout URIs were interrupted, or another error occurred in scheduling the requests for execution.
 BACKCHANNEL_LOGOUT_REQUESTS_INVOKE_ALL_FAILED.useraction=Send another logout request to retry the back-channel logout. For more information, see the error included in the message.
 
 ERROR_BUILDING_OR_SENDING_BACKCHANNEL_LOGOUT_REQUEST=CWWKS1951E: The {0} OpenID Connect provider encountered an error while building or sending a back-channel logout request to {1}: {2}
 ERROR_BUILDING_OR_SENDING_BACKCHANNEL_LOGOUT_REQUEST.explanation=The HTTP connection encountered an error, or the connection failed.
 ERROR_BUILDING_OR_SENDING_BACKCHANNEL_LOGOUT_REQUEST.useraction=Send another logout request to retry the back-channel logout. For more information, see the error included in the message.
+
+ERROR_BUILDING_LOGOUT_TOKEN_BASED_ON_ID_TOKEN=CWWKS1952W: A back-channel logout token that is based on a cached ID token for the {0} client cannot be created due to the following error: {1}
+ERROR_BUILDING_LOGOUT_TOKEN_BASED_ON_ID_TOKEN.explanation=An error occurred in building the logout token string. Alternatively, the issuer of the cached ID token might not match this OpenID Connect provider, or the cached ID token might be missing some claims that are required.
+ERROR_BUILDING_LOGOUT_TOKEN_BASED_ON_ID_TOKEN.useraction=For more information, see the error included in the message.
+
+ERROR_BUILDING_LOGOUT_TOKEN_BASED_ON_ID_TOKEN_CLAIMS=CWWKS1953E: A back-channel logout token that is based on a cached ID token with claims [{0}] for the {1} client cannot be created due to the following error: {2}
+ERROR_BUILDING_LOGOUT_TOKEN_BASED_ON_ID_TOKEN_CLAIMS.explanation=An error occurred while building the logout token string. Alternatively, the issuer of the cached ID token might not match this OpenID Connect provider, or the cached ID token might be missing some claims that are required.
```

Comment:
explanation:
An error occurred in building the logout token string. Alternatively, the issuer of the cached ID token might not match this OpenID Connect provider, or the cached ID token might be missing some claims that are required.